tests/types/mlflow-test.json

```diff
@@ -260,6 +260,104 @@
+    },
+    {
+      "description": "Valid MLflow package with name and version only",
+      "test_group": "base",
+      "test_type": "validation",
+      "input": {
+        "type": "mlflow",
+        "namespace": null,
+        "name": "creditfraud",
+        "version": "3",
+        "qualifiers": null,
+        "subpath": null
+      },
+      "expected_messages": []
+    },
+    {
+      "description": "Valid MLflow package with repository_url qualifier",
+      "test_group": "base",
+      "test_type": "validation",
+      "input": {
+        "type": "mlflow",
+        "namespace": null,
+        "name": "creditfraud",
+        "version": "3",
+        "qualifiers": {
+          "repository_url": "https://westus2.api.azureml.ms/mlflow/v1.0/subscriptions/a50f2011-fab8-4164-af23-c62881ef8c95/resourceGroups/TestResourceGroup/providers/Microsoft.MachineLearningServices/workspaces/TestWorkspace"
+        },
+        "subpath": null
+      },
+      "expected_messages": []
+    },
+    {
+      "description": "Valid MLflow package with model_uuid and run_id qualifiers",
+      "test_group": "base",
+      "test_type": "validation",
+      "input": {
+        "type": "mlflow",
+        "namespace": null,
+        "name": "trafficsigns",
+        "version": "10",
+        "qualifiers": {
+          "model_uuid": "36233173b22f4c89b451f1228d700d49",
+          "run_id": "410a3121-2709-4f88-98dd-dba0ef056b0a",
+          "repository_url": "https://adb-5245952564735461.0.azuredatabricks.net/api/2.0/mlflow"
+        },
+        "subpath": null
+      },
+      "expected_messages": []
+    },
+    {
+      "description": "Invalid MLflow package with namespace (not allowed)",
+      "test_group": "advanced",
+      "test_type": "validation",
+      "input": {
+        "type": "mlflow",
+        "namespace": "azure",
+        "name": "creditfraud",
+        "version": "3",
+        "qualifiers": null,
+        "subpath": null
+      },
+      "expected_messages": [
+        "Namespace is prohibited for purl type: 'mlflow'"
+      ]
+    },
+    {
+      "description": "MLflow package with unsupported qualifier",
+      "test_group": "advanced",
+      "test_type": "validation",
+      "input": {
+        "type": "mlflow",
+        "namespace": null,
+        "name": "creditfraud",
+        "version": "3",
+        "qualifiers": {
+          "foo": "bar"
+        },
+        "subpath": null
+      },
+      "expected_messages": null
+    },
+    {
+      "description": "MLflow package with unsupported qualifier",
+      "test_group": "base",
+      "test_type": "validation",
+      "input": {
+        "type": "mlflow",
+        "namespace": null,
+        "name": "creditfraud",
+        "version": "3",
+        "qualifiers": {
+          "foo": "bar"
+        },
+        "subpath": null
+      },
+      "expected_messages": [
+        "Invalid qualifiers found: foo. Allowed qualifiers are: model_uuid, repository_url, run_id"
```

Comment:
I don't think it's right to reject unknown qualifiers. It provides little value while creating problems for extensibility and evolution.

Reply:
@matt-phylum agreed. @TG1999 we cannot reject unknown qualifiers, as a principle. Please update accordingly